frontend/src/pages/LandlordPage.tsx

```diff
@@ -28,13 +31,63 @@ const reviews = [
 
 const LandlordPage = (): ReactElement => {
   const { landlordId } = useParams<Record<string, string | undefined>>();
+  const features = ['Parking', 'Heating', 'Trash removal', 'Snow plowing', 'Maintenance'];
```

Comment:
Instead of having separate consts for each individual set of data in a landlord page, maybe make a type for this landlord data and use a `useState<LandlordData>` hook with these default values instead. This will make it easier when we're getting data from the DB.